docker/proxy-config/nginx.conf

```diff
@@ -0,0 +1,42 @@
+server {
+    listen       80;
+    server_name  localhost;
+    client_max_body_size 100M;
+    return 301 https://$host$request_uri;
+}
+
+server {
+	listen 443 ssl;
+	listen [::]:443 default_server;
+ssl_certificate /etc/ssl/safu.crt;
```

Comment:
Update README.MD how to generate these certificates:

```
openssl req -x509 -out localhost.crt -keyout localhost.key \
  -newkey rsa:2048 -nodes -sha256 \
  -subj '/CN=localhost' -extensions EXT -config <( \
   printf "[dn]\nCN=localhost\n[req]\ndistinguished_name = dn\n[EXT]\nsubjectAltName=DNS:localhost\nkeyUsage=digitalSignature\nextendedKeyUsage=serverAuth")
```

Something like this worked for me well..